app/src/main/java/edu/uc/groupproject/covid19tracker/MainActivity.kt

```diff
@@ -6,7 +6,6 @@ import androidx.fragment.app.FragmentTransaction
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.groupproject.covid19tracker.ui.main.ByCountryFragment
 import edu.uc.groupproject.covid19tracker.ui.main.MainFragment
-import edu.uc.groupproject.covid19tracker.ui.main.MainViewModel
```

Comment:
Wasn't being used

Reply:
Control-Alt-O in Android Studio to organize imports automatically.  :)